### PR TITLE
Fixes #8701: Add rudder development environment to rtf

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/8701

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

File: platforms/dev.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "default":{ "run-with": "vagrant", "rudder-version": "3.2", "system": "debian8", "inventory-os": "debian" },
+  "server": { "rudder-setup": "dev" },
```
Comment (peckpeck) [inline platforms/dev.json]: I would prefer a more explicit name like server-dev

**Comment by peckpeck** [comment]
Looks very good.
How do you configure eclipse to use it ?

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

File: README.md [hunk 2]
```diff
@@ -125,3 +125,18 @@ Everything on host is done through the Host class in rtf.
...
+
+Using dev environment
+------------------------
+
+It's possible to create a development environment with rtf.
+We provide a platform to launch your environment, just run: 
+```
+./rtf platform setup dev
+```
+You can also add a dev server to any platform by setting it's "rudder-setup" to "dev-server" (like we did in dev.json)
```
Comment (peckpeck) [inline README.md]: *its* "rudder-setup" *parameter*
like we *do* in

File: README.md [hunk 3]
```diff
@@ -125,3 +125,18 @@ Everything on host is done through the Host class in rtf.
...
+
+To use config file provided in your Eclipse, just add this line the project run configuration arguments:
```
Comment (peckpeck) [inline README.md]: To use the provided configuration file in

File: README.md [hunk 4]
```diff
@@ -125,3 +125,18 @@ Everything on host is done through the Host class in rtf.
...
+```
+-Drudder.configFile=/home/vincent/vmVagrant/rudder-dev-vagrant/config/configuration.properties
```
Comment (peckpeck) [inline README.md]: This is not a rudder-test/dev/ directory

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Pull request merged**